corehq/motech/fhir/views.py

```diff
@@ -0,0 +1,29 @@
+from django.http import JsonResponse
+
+from corehq import toggles
+from corehq.apps.domain.decorators import require_superuser, login_and_domain_required
+from corehq.form_processor.exceptions import CaseNotFound
+from corehq.form_processor.interfaces.dbaccessors import CaseAccessors
+from corehq.motech.fhir.models import FHIRResourceType, build_fhir_resource
+
+
+@login_and_domain_required
+@require_superuser
+@toggles.FHIR_INTEGRATION.required_decorator()
+def get_view(request, domain, resource_type, resource_id):
+    try:
+        case = CaseAccessors(domain).get_case(resource_id)
+        if case.is_deleted:
+            return JsonResponse(status=400, data={'message': f"Resource with ID {resource_id} was removed"})
+    except CaseNotFound:
+        return JsonResponse(status=400, data={'message': f"Could not find resource with ID {resource_id}"})
+
+    if case.type not in (FHIRResourceType.objects.filter(domain=domain, name=resource_type).
+                         values_list('case_type__name', flat=True)):
```

Comment:
```suggestion
    fhir_version = [v[0] for v in FHIR_VERSIONS if v[1] == version_name][0]
    if not FHIRResourceType.objects.filter(
        domain=domain,
        name=resource_type,
        fhir_version=fhir_version,
        case_type__name=case.type,
    ).exists():
```

Reply:
could not commit directly from here since the view arguments would need to be changed as well, so made a commit.
Thanks for reminding of "exists"